slackhealthbot/domain/modelmappers/repositorytocore/oauthfitbit.py

```diff
@@ -0,0 +1,14 @@
+from slackhealthbot.core.models import OAuthFields
+from slackhealthbot.data.repositories import fitbitrepository
+
+
+def repository_oauth_to_core_oauth(
```

Comment:
Will maybe refactor the archi (again 😅 🙄 ), and use domain models from use cases and repositories. That should hopefully mean fewer mapper classes to maintain.